tools/esdlcomp/esdlcomp.cpp

```diff
@@ -1109,8 +1107,22 @@ ESDLcompiler::ESDLcompiler(const char * sourceFile, bool generatefile, const cha
     yyin = fopen(sourceFile, "rt");
     if (!yyin)
     {
-        fprintf(stderr, "Fatal Error: Cannot read %s\n",sourceFile);
-        exit(1);
+        fprintf(stderr, "Could not load: %s...\n",sourceFile);
+        StringBuffer alternateExtFilename(srcDir);
+        alternateExtFilename.append(name.str());
+
+        if (stricmp(ext.str(), ESDL_FILE_EXTENTION)==0)
+            alternateExtFilename.append(LEGACY_FILE_EXTENTION);
+        else
+            alternateExtFilename.append(ESDL_FILE_EXTENTION);
+
+        fprintf(stderr, "Attempting to re-load with alternate extension: %s\n",alternateExtFilename.str());
```

Comment:
I think we need to know if we got here via include or this is the top level file specified on command line.

If this is an include we should check both extensions silently.
If this is the file specified on command line we should use exactly what the user specified.

In the future we may want to allow the full name to be specified in the Include statement.